### PR TITLE
tests: Add tests lib common script

### LINKED ISSUE #7113: tests: Add tests lib common script

### PR DESCRIPTION
**Pull request by GabyCT**
This PR adds the test lib common script that is going to be used for kata containers metrics.

Fixes #7113

### CODE REVIEW

**Review by dborquez — APPROVED** [review]
LGTM, thank you @GabyCT

File: tests/metrics/lib/common.bash [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+# system to a 'known state' as much as possible
+metrics_onetime_init()
+{
+	# The onetime init must be called once, and only once
+	if [ ! -z "$onetime_init_done" ]; then
+		die "onetime_init() called more than once"
+	fi
+
+	# Restart services
+	restart_docker_service
```
Comment (dborquez) [inline tests/metrics/lib/common.bash:149]: @GabyCT Are we still using docker on metrics?
if not, could you please remove `restart_docker_service`?

**Review by dborquez — CHANGES_REQUESTED** [review]

File: tests/metrics/lib/common.bash (hunk 1 above)
Comment (GabyCT) [reply tests/metrics/lib/common.bash]: @dborquez changes applied

**Review by dborquez — APPROVED** [review]
@GabyCT LGTM, thank you

File: tests/metrics/lib/common.bash [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+# Set variables to reasonable defaults if unset or empty
+CTR_EXE="${CTR_EXE:-ctr}"
+DOCKER_EXE="${DOCKER_EXE:-docker}"
+CTR_RUNTIME="${CTR_RUNTIME:-io.containerd.kata.v2}"
+RUNTIME="${RUNTIME:-containerd-shim-kata-v2}"
+KATA_HYPERVISOR="${KATA_HYPERVISOR:-qemu}"
+TEST_REPO="${TEST_REPO:-github.com/kata-containers/tests}"
+JSON_HOST="${JSON_HOST:-}"
+
+KSM_BASE="/sys/kernel/mm/ksm"
```
Comment (chavafg) [inline tests/metrics/lib/common.bash:24]: I remember the KSM values here and below were used by the ksm-throttler, which I believe is no longer supported, so it should be fine to delete them if we don't plan to test KSM, but would like to see if someone else can confirm.
For reference: https://github.com/kata-containers/kata-containers/issues/1073

Comment (GabyCT) [reply tests/metrics/lib/common.bash:24]: @chavafg we are not using ksm-throttler for Kata but we are using some memory tests that are only using ksm

Comment (chavafg) [reply tests/metrics/lib/common.bash:24]: ok

**Review by chavafg — APPROVED** [review]

**Pull request merged**